examples/compute_polar_coordinates.py

```diff
@@ -0,0 +1,362 @@
+"""Express 2D vectors in polar coordinates
+============================================
+
+Compute a vector representing head direction and express it in polar
+coordinates.
+"""
+
+# %%
+# Imports
+# -------
+
+# For interactive plots: install ipympl with `pip install ipympl` and uncomment
+# the following line in your notebook
+# %matplotlib widget
+import numpy as np
+from matplotlib import pyplot as plt
+
+from movement import sample_data
+from movement.io import load_poses
+from movement.utils.vector import cart2pol, pol2cart
+
+# %%
+# Load sample dataset
+# ------------------------
+# In this tutorial, we will use a sample dataset with a single individual
+# (a mouse) and six keypoints.
+
+ds_path = sample_data.fetch_dataset_paths(
+    "SLEAP_single-mouse_EPM.analysis.h5"
+)["poses"]
+ds = load_poses.from_sleap_file(ds_path, fps=None)  # force time_unit = frames
+
+print(ds)
+print("-----------------------------")
+print(f"Individuals: {ds.individuals.values}")
+print(f"Keypoints: {ds.keypoints.values}")
+
+
+# %%
+# The loaded dataset ``ds`` contains two data variables:``position`` and
+# ``confidence``. Both are stored as data arrays. In this tutorial, we will
+# use only ``position``:
+position = ds.position
+
+
+# %%
+# Compute head vector
+# ---------------------
+# To demonstrate how polar coordinates can be useful in behavioural analyses,
+# we will compute the head vector of the mouse.
+#
+# We define it as the vector from the midpoint between the ears to the snout.
+
+# compute the midpoint between the ears
+midpoint_ears = 0.5 * (
+    position.sel(keypoints="left_ear") + position.sel(keypoints="right_ear")
+)
+
+# compute the head vector
+head_vector = position.sel(keypoints="snout") - midpoint_ears
+
+# drop the keypoints dimension
+head_vector = head_vector.drop_vars("keypoints")
+
+# %%
+# Visualise the head trajectory
+# ---------------------------------
+# We can plot the data to check that our computation of the head vector is
+# correct.
+#
+# We can start by plotting the trajectory of the midpoint between the ears. We
+# will refer to this as the head trajectory.
+
+fig, ax = plt.subplots(1, 1)
+mouse_name = ds.individuals.values[0]
+
+sc = ax.scatter(
+    midpoint_ears.sel(individuals=mouse_name, space="x"),
+    midpoint_ears.sel(individuals=mouse_name, space="y"),
+    s=15,
+    c=midpoint_ears.time,
+    cmap="viridis",
+    marker="o",
+)
+
+ax.axis("equal")
+ax.set_xlabel("x (pixels)")
+ax.set_ylabel("y (pixels)")
+ax.invert_yaxis()
+ax.set_title(f"Head trajectory ({mouse_name})")
+fig.colorbar(sc, ax=ax, label=f"time ({ds.attrs['time_unit']})")
+fig.show()
+
+# %%
+# We can see that the majority of the head trajectory data is within a
+# cruciform shape. This is because the dataset is of a mouse moving on an
+# [Elevated Plus Maze](https://en.wikipedia.org/wiki/Elevated_plus_maze).
```

Comment:
This link doesn't render as you'd expect, because the markup languages for these text blocks is rst not md. Use the appropriate sphinx rst syntax for hyperlinks, see https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html